src/scripting_api/scripting.js

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2020 Mozilla Foundation
```

Comment:
This file doesn't seem to have a lot of purpose. Can't we just merge this with the code generator for simplicity.